cache-config/t3c-apply/torequest/cmd.go

```diff
@@ -333,6 +336,22 @@ func checkRefs(cfg config.Cfg, cfgFile []byte, filesAdding []string) error {
 	return nil
 }
 
+//checkCert checks the validity of the ssl certificate
+func checkCert(c []byte) error {
+	block, _ := pem.Decode(c)
+	cert, err := x509.ParseCertificate(block.Bytes)
+	if err != nil {
+		return err
```

Comment:
Nitpick: context would make it easier to figure out what went wrong when this happens, e.g. `return errors.New("parsing certificate: " + err.Error())`